index.js

```diff
@@ -53,13 +63,17 @@ NYC.prototype._createInstrumenter = function () {
 NYC.prototype.addFile = function (filename, returnImmediately) {
   var instrument = true
   var relFile = path.relative(this.cwd, filename)
+  var include
 
-  // only instrument a file if it's not on the exclude list.
   for (var i = 0, exclude; (exclude = this.exclude[i]) !== undefined; i++) {
+    include = this.include[i]
```

Comment:
I see a potential problem here. If the exclude list is shorter than the include list, it will not get through all the includes.